{{cookiecutter.repository_name}}/{{cookiecutter.package_name}}/model/standardized_image.py

```diff
@@ -28,7 +28,7 @@ def from_image(image: Image.Image):
         return StandardizedImage(data=standardize(resize(image=image)))
 
     @staticmethod
-    def from_example(example: problem.Example):
+    def from_example(example):
```

Comment:
Did not think about the issue of not being able to type hint. I suppose it's not major but it's a little sad :cry:

Reply:
Agreed. I think this says something about the method though. Maybe it's not supposed to exist?